### PR TITLE
Core/Spell Implement two new spell for Demon hunter.

### PR DESCRIPTION
**Pull request by HannibalRoG**
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

**Changes proposed:**
Implement two new spell for DemonHunter (the first iiijjaaa)
- Chaos strike
- Demon blades

**Target branch(es):** 3.3.5/master

- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)
I cant find any.

**Tests performed:** (Does it build, tested in-game, etc.)
Compiled and tested in game, they look good for my inexpert eye.

### CODE REVIEW

**Comment by Seyden** [comment]
Demon Blades doesn't need a script anymore.. they already changed it with trigger spell.. you're pretty out to date

**Comment by Seyden** [comment]
And the proc ex critical is missing for chaos strike

File: sql/updates/world/master/RoG_2017_03_09_00_world.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_dh_chaos_strike';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (197125, 'spell_dh_chaos_strike');
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_dh_demon_blades';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (203555, 'spell_dh_demon_blades');
```
Comment (Keader) [inline sql/updates/world/master/RoG_2017_03_09_00_world.sql]:
```sql

DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_dh_chaos_strike','spell_dh_demon_blades');
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
(197125, 'spell_dh_chaos_strike'),
(203555, 'spell_dh_demon_blades');
```

File: src/server/scripts/Spells/spell_dh.cpp [hunk 2]
```diff
@@ -16,11 +16,98 @@
...
+        return new spell_dh_demon_blades_AuraScript();
+    }
+};
+ 
 void AddSC_demon_hunter_spell_scripts()
 {
-}
+    new spell_dh_chaos_strike();
+    new spell_dh_demon_blades();
+}
```
Comment (Keader) [inline src/server/scripts/Spells/spell_dh.cpp]: Missing new line

**Comment by ChazyTheBest** [comment]
There's already a PR for [Chaos Strike](17656).

**Comment by HannibalRoG** [comment]
I think that PR was abandoned, we will try to keep it up and integrate the better of the two chaos strike.

**Comment by P-Kito** [comment]
Read Seyden's comments, he says you are working on outdated DBC/DB2

**Comment by HannibalRoG** [comment]
Ok, we see the point, we update PR with only Chaos Strike.

**Comment by ChazyTheBest** [comment]
@HannibalRoG @Ovahlord seems to be on vacation, but I don't think its abandoned.

**Comment by HannibalRoG** [comment]
Ok, close this PR then.

**Pull request closed without merging**